nav2_recoveries/src/spin.cpp

```diff
@@ -49,50 +49,50 @@ Spin::~Spin()
 
 Status Spin::onRun(const std::shared_ptr<const SpinAction::Goal> command)
 {
-  double yaw, pitch, roll;
-  tf2::getEulerYPR(command->target.quaternion, yaw, pitch, roll);
+  double pitch, roll;
+  tf2::getEulerYPR(command->target.quaternion, command_yaw_, pitch, roll);
```

Comment:
If not supporting multiple rotations, maybe add a check that the commanded velocity is within a single rotation.